Sources/AppcuesKit/Presentation/UI/ExperienceData.swift

```diff
@@ -126,7 +131,7 @@ extension ExperienceData {
                 case .single(let value):
                     return value
                 case .multi(let values, _):
-                    return values.joined(separator: ",")
+                    return values.joined(separator: "\n")
```

Comment:
Note I snuck in the change discussed this morning to separate values with newlines rather than commas